src/internal/packager/git/clone.go

```diff
@@ -46,7 +46,7 @@ func (g *Git) clone(gitDirectory string, gitURL string, onlyFetchRef bool) (*git
 		return repo, git.ErrRepositoryAlreadyExists
 	} else if err != nil {
 		message.Debugf("Failed to clone repo: %s", err.Error())
-		message.Infof("Falling back to host git for %s", gitURL)
+		g.Spinner.Updatef("Falling back to host git for %s", gitURL)
```

Comment:
If we switch these back to spinner updates, I would like to the gitURL in the debug message that is right above it since that will go away with the spinner now.

Reply:
I suppose that's fair--though the log file will still have it even if not in debug :-)